pineappl/src/grid.rs

```diff
@@ -1607,8 +1613,13 @@ impl Grid {
                             .grid_axes
                             .muf2_grid
                             .iter()
-                            .position(|&q2| approx_eq!(f64, q2, src_q2, ulps = 64))
-                            .unwrap()
+                            .position(|&q2| approx_eq!(f64, q2, eko_info.xif * eko_info.xif * src_q2, ulps = 64))
```

Comment:
If the square is intentional (as I guess) please use
```suggestion
                            .position(|&q2| approx_eq!(f64, q2, eko_info.xif.pow(2) * src_q2, ulps = 64))
```

It's a bit more explicit.

Reply:
In Rust you don't have `pow` but instead `powi` for integer exponents and `powf` for real exponents. However, if you use `powi` clippy will probably complain about the manual square being faster. I'd say it's a non-issue for the time being.

Reply:
Just checked, this compiles:

```rust
fn main() {
    let a: i64 = 2;
    println!("2^3 = {}", a.pow(3));
}
```

Reply:
https://doc.rust-lang.org/stable/std/?search=pow

Reply:
then I keep as is (and be consistent with the [existing code](https://github.com/N3PDF/pineappl/blob/3fd98031fb4e2fbef7d5e77b1d900c8aed31d7a5/pineappl/src/grid.rs#L1573))